src/Lean/Elab/PreDefinition/WF/Rel.lean

```diff
@@ -67,7 +67,14 @@ def elabWFRel (preDefs : Array PreDefinition) (unaryPreDefName : Name) (fixedPre
     for (d, mvarId) in subgoals, element in wf, preDef in preDefs do
       let mvarId ← unpackUnary preDef fixedPrefixSize mvarId d element
       mvarId.withContext do
+        let errorMsgHeader? := if preDefs.size > 1 then
+          "The termination argument types differ for the different functions, or depends on the " ++
```

Comment:
Subject-verb agreement:
```suggestion
          "The termination argument types differ for the different functions, or depend on the " ++
```

Reply:
So I had it right initially (https://github.com/leanprover/lean4/pull/3414/commits/bddb56d69bebee3ad6fadcf55942b058d89ed8a1) 🤦‍♂️